hoi/metrics/oinfo_zerolag.py

```diff
@@ -0,0 +1,123 @@
+from math import comb
+import jax
+import jax.numpy as jnp
+import numpy as np
+from scipy.special import ndtri
+import logging
+
+logger = logging.getLogger("frites")
+from hoi.oinfo.conn_oinfo_jax import combinations
+from hoi.oinfo.conn_oinfo_jax import oinfo_mmult
+
+
+def ctransform(x):  # frites.core
+    xr = np.argsort(np.argsort(x)).astype(float)
+    xr += 1.0
+    xr /= float(xr.shape[-1] + 1)
+    return xr
+
+
+def copnorm_1d(x):  # frites.core
+    assert isinstance(x, np.ndarray) and (x.ndim == 1)
+    return ndtri(ctransform(x))
+
+
+def copnorm_nd(x, axis=-1):  # frites.core
+    assert isinstance(x, np.ndarray) and (x.ndim >= 1)
+    return np.apply_along_axis(copnorm_1d, axis, x)
+
+
+def conn_oinfo_jax(
+    data,
+    y=None,
+    variables=None,
+    features=None,
+    minsize=3,
+    maxsize=5,
+    sort=True,
```

Comment:
Is the sorting used somewhere?

Reply:
Yes. The function `combinations` is called within `oinfo_zerolag` which has a parameter `sort` to create a lexicographic order of the regions.

Reply:
Indeed. We could remove the `feature` input of the `combination` function, such as the sorting. We can do the sorting later, once we identify significant multiplets we could build the sorted name of the multiplets.